components/admin/requests/forms/new-applications/GuardianInformationForm.tsx

```diff
@@ -0,0 +1,203 @@
+
+/**
+ * GuardianInformationForm Component for allowing users to edit guardian information.
+ *
+ * @param {GuardianInformation} permitHolderInformation Data Structure that holds all guardian information for a client request.
+ * @param onChange Function that uses the updated values from form.
+ */
+export default function GuardianInformationForm({
+  guardianInformation,
+  onChange,
+  onFileUpload,
+}: GuardianInformationFormProps) {
+  return (
+    <>
+      {/* Personal Information Section */}
+      <Box paddingBottom="32px">
+        <Text as="h3" textStyle="heading" paddingBottom="24px">
+          {'Guardian/POA Information'}
+        </Text>
+        <Stack direction="row" spacing="20px">
+          <FormControl isRequired>
+            <FormLabel>{'First name'}</FormLabel>
+            <Input
+              value={guardianInformation.firstName}
+              onChange={event =>
+                onChange({
+                  ...guardianInformation,
+                  firstName: event.target.value,
+                })
+              }
+            />
+          </FormControl>
+
+          <FormControl>
+            <FormLabel>{'Middle name'}</FormLabel>
+            <Input
+              value={guardianInformation.middleName}
+              onChange={event =>
+                onChange({
+                  ...guardianInformation,
+                  lastName: event.target.value,
+                })
+              }
+            />
+          </FormControl>
+
+          <FormControl isRequired>
+            <FormLabel>{'Last name'}</FormLabel>
+            <Input
+              value={guardianInformation.lastName}
+              onChange={event =>
+                onChange({
+                  ...guardianInformation,
+                  lastName: event.target.value,
+                })
+              }
+            />
+          </FormControl>
+        </Stack>
+
+        <Stack direction="row" spacing="20px" paddingTop="20px">
+          <FormControl isRequired>
+            <FormLabel>{'Phone number'}</FormLabel>
+            <Input
+              value={guardianInformation.phone}
+              type="tel"
+              onChange={event =>
+                onChange({
+                  ...guardianInformation,
+                  phone: event.target.value,
+                })
+              }
+            />
+            <FormHelperText color="text.seconday">{'Example: 000-000-0000'}</FormHelperText>
+          </FormControl>
+
+          <FormControl isRequired>
+            <FormLabel>{'Relationship to applicant'}</FormLabel>
+            <Input
+              value={guardianInformation.guardianRelationship || ''}
+              onChange={event =>
+                onChange({
+                  ...guardianInformation,
+                  guardianRelationship: event.target.value,
+                })
+              }
+            />
+          </FormControl>
+        </Stack>
+      </Box>
+
+      <Divider borderColor="border.secondary" />
```

Comment:
```suggestion
      <Divider />
```
Don't need that prop now, since it's been added to the default props in `tools/theme/components/divider.ts`